src/Core/Services/Implementations/AzureQueuePushNotificationService.cs

```diff
@@ -128,11 +129,7 @@ public async Task PushLogOutAsync(Guid userId, bool excludeCurrentContext = fals
 
     private async Task PushUserAsync(Guid userId, PushType type, bool excludeCurrentContext = false)
     {
-        var message = new UserPushNotification
-        {
-            UserId = userId,
-            Date = DateTime.UtcNow
-        };
+        var message = new UserPushNotification { UserId = userId, Date = DateTime.UtcNow };
```

Comment:
Nitpick: You can inject `TimeProvider` in the `AzureQueuePushNotificationService`.

https://andrewlock.net/exploring-the-dotnet-8-preview-avoiding-flaky-tests-with-timeprovider-and-itimer/

Reply:
I was planning on adding tests at least to affected changes, once i get some positive feedback. I can't look at this 0% coverage, it's hideous.
So far i have been using "close to" date matching: https://github.com/bitwarden/server/pull/4923/files#diff-2167df5f2928366292863c830a36efa014652ebf0af11204c238c6dc139c824bR57
I will consider this for, thank you!